articles/governance/policy/how-to/remediate-resources.md

```diff
@@ -89,15 +131,19 @@ $policyDef = Get-AzPolicyDefinition -Id '/providers/Microsoft.Authorization/poli
 # Get the reference to the resource group
 $resourceGroup = Get-AzResourceGroup -Name 'MyResourceGroup'
 
-# Create the assignment using the -Location and -AssignIdentity properties
-$assignment = New-AzPolicyAssignment -Name 'sqlDbTDE' -DisplayName 'Deploy SQL DB transparent data encryption' -Scope $resourceGroup.ResourceId -PolicyDefinition $policyDef -Location 'westus' -AssignIdentity
+# Get the existing user assigned managed identity ID
+$userassignedidentity = New-AzUserAssignedIdentity -ResourceGroupName $rgname -Name $userassignedidentityname -Location $location
+$userassignedidentityid = $userassignedidentity.Id
+
+# Create the assignment using the -Location and -Identity properties
+$assignment = New-AzPolicyAssignment -Name 'sqlDbTDE' -DisplayName 'Deploy SQL DB transparent data encryption' -Scope $resourceGroup.ResourceId -PolicyDefinition $policyDef -Location 'westus' -IdentityType "UserAssigned" -IdentityId $userassignedidentityid
 ```
 
 The `$assignment` variable now contains the principal ID of the managed identity along with the
 standard values returned when creating a policy assignment. It can be accessed through
 `$assignment.Identity.PrincipalId`.
```

Comment:
$assignment.Identity.PrincipalId will give the principal ID for a system assigned id. For a user assigned id, we would access the principal id through $userassignment1.Identity.UserAssignedIdentities[$userassignedidentityid].PrincipalId

Reply:
addressed.